src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -104,33 +103,26 @@
     private Set<Vector3i> detectedBlocks = new HashSet<>();
 
     /**
-    * This set the local player for the tests
+    * This sets the local player for the tests
     * The dummy local player will make the system work and does not throw a null exception
-    * @param set the local player that going to be set through the test method
```

Comment:
I've explained in some other comment the types of javadocs for accessors. For now, let's have the param. However, change it to something descriptive, instead of removing it:
`@param set A LocalPlayer wwhose location component is used`